labgrid/driver/sshdriver.py

```diff
@@ -357,7 +370,7 @@ def scp(self, *, src, dst):
                 "-o", f"ControlPath={self.control.replace('%', '%%')}",
                 src, dst,
         ]
-        
+
```

Comment:
Seems unrelated.

Reply:
<img width="502" alt="lg" src="https://github.com/labgrid-project/labgrid/assets/3738458/b26c92c3-462c-414c-9b9a-5fc399486138">

My vim plugin automatically delete useless blank in a empty line when save. Ok, I disable my vim plugin and leave this line ASIS.